documentation/directory-structure.md

```diff
@@ -30,140 +30,102 @@ You may be also interested in following:
+You can find details on the name and contents of `<typeSpecSrc>` folders in [TypeSpec directory structure].
+You can learn more about TypeSpec at [aka.ms/azsdk/typespec] and [aka.ms/typespec].
 
-``` yaml
-/<typeSpecFolder>1/...
-/<typeSpecFolder>2/... # multiple '<typeSpecFolder>' folders
+### `resource-manager/<RPNS>` folder
 
-/resource-manager/README.md
-/resource-manager/<RPNS>/stable/<apiVersion1>/*.json
-/resource-manager/<RPNS>/stable/<apiVersion1>/examples/*.json
-                               /<apiVersion2>/ ... # multiple '<apiVersion>' folders
-/resource-manager/<RPNS>/preview/<apiVersion3-preview>/*.json
-/resource-manager/<RPNS>/preview/<apiVersion3-preview>/examples/*.json
-                                /<apiVersion4-preview>/ ... # multiple '<apiVersion-preview>' folders
+The `<azureTeam>/resource-manager/<RPNS>` is a folder corresponding to **Resource Provider (RP) namespace**.
 
-/data-plane/README.md
-/data-plane/<dataPlaneSubfolder1>/stable/<apiVersion5>/*.json
-/data-plane/<dataPlaneSubfolder1>/stable/<apiVersion5>/examples/*.json
-                                        /<apiVersion6>/ ... # multiple '<apiVersion>' folders
-/data-plane/<dataPlaneSubfolder1>/preview/<apiVersion7-preview>/*.json
-/data-plane/<dataPlaneSubfolder1>/preview/<apiVersion7-preview>/examples/*.json
-                                         /<apiVersion8-preview>/ ... # multiple '<apiVersion-preview>' folders
-           /<dataPlaneSubfolder2>/ ... # multiple '<dataPlaneSubfolder>' folders
+An example RPNS is `Microsoft.Automation`. A list of RPs can be found in the [Resource Provider list].
 
-```
+This folder corresponds to a `service group` (see [glossary]) for all ARM services owned by the team.
 
-As a specific example of the above, consider [`specification/confidentialledger`] `<azureTeam>` which has the following structure:
+An `<RPNS>` folder has one or more children `<service>` folders corresponding to the services belonging 
+to the `service group` represented by the `<RPNS>` folder.
 
-``` yaml
+For example, [`specification/containerservice/resource-manager/Microsoft.ContainerService/aks`] 
+is a folder for the `aks` service within the `Microsoft.ContainerService` ARM Resource Provider namespace.
 
-# ===== <typeSpecFolder>s
+> [!NOTE]
+> Many Azure teams that have one ARM service instead of rooting it in `specification/<azureTeam>/resource-manager/<RPNS>/<service>` root it
+> in `specification/<azureTeam>/resource-manager`. This is legacy, deprecated structure and is not allowed going forward.
 
-/Microsoft.CodeTransparency/
-/Microsoft.ManagedCcf/
+In addition, the `<RPNS>` folder may contain `common-types` child folder that usually is of form `common-types/v<versionInt>/common.json` and contains types
+shared across API versions. E.g. [`Microsoft.Compute/common-types`].
 
-# ===== data-plane
+### `data-plane/<namespace>` folders
```

Comment:
namespace?
I won't point out future uses of namespace but I still question them.

Reply:
Let's discuss here:
https://github.com/Azure/azure-rest-api-specs/pull/28929#discussion_r1625472887